tests/ssl-opt.sh

```diff
@@ -5459,35 +5515,31 @@ run_test    "DTLS fragmenting: gnutls server, DTLS 1.0" \
             -c "fragmenting handshake message" \
             -C "error"
 
-# gnutls-cli always tries IPv6 first, and doesn't fall back to IPv4 with DTLS
-requires_ipv6
 requires_config_enabled MBEDTLS_SSL_PROTO_DTLS
```

Comment:
I think we need to have a comment here that explains and justifies the use of `--insecure` here and below.

Reply:
@sbutcher-arm I don't mind changing this here, but the change was introduced by @mpg in #1939, and if your feedback on that PR leads to a 'fork' of #1939 being created anyway, I think it's better to add the change you requested here in that new PR, too.

Reply:
Please liaise with @mpg and do what you think is best. I'd prefer it if you could do it, as I think you'll have more time than @mpg, but I'll leave it to you to decide. Just do it as a new PR, so I can get this one in without more rework.